src/panel/category/CategoryPanel.jsx

```diff
@@ -102,26 +96,23 @@ export default class CategoryPanel extends React.Component {
       window.map.mb.flyTo({ center: [2.35, 48.85], zoom: 12 });
     } else if (currentZoom < 12) { // Zoom < 12: zoom up to zoom 12
       window.map.mb.flyTo({ zoom: 12 });
-    } else if (currentZoom > 16) { // Zoom > 16: dezoom to zoom 16
-      window.map.mb.flyTo({ zoom: 16 });
```

Comment:
This rule could actually still be useful to avoid very large zoom out (maybe for z>18 only ?)


I understand that we want to avoid multiple zoom changes, but they can still happen in some cases with the previous rule (z=10 :arrow_right: z=12 :arrow_right: extended bbox to z=11)

Reply:
I restored the rule but changed it to z>18 as you proposed.